src/util/XDRCereal.h

```diff
@@ -47,43 +47,14 @@ cereal_override(cereal::JSONOutputArchive& ar, const xdr::opaque_vec<N>& s,
     xdr::archive(ar, stellar::binToHex(stellar::ByteSlice(s.data(), s.size())),
                  field);
 }
-void
-cereal_override(cereal::JSONOutputArchive& ar, const stellar::PublicKey& s,
-                const char* field)
-{
-    xdr::archive(ar, stellar::KeyUtils::toStrKey<stellar::PublicKey>(s), field);
-}
+void cereal_override(cereal::JSONOutputArchive& ar, const stellar::PublicKey& s,
```

Comment:
Nitpick: also missing whitespace here. My general principle on this is that an implementation is always sandwiched between whitespace. Definitions can be stacked together if they are overloads or are otherwise very closely related, although this is optional.

Reply:
Noted! Fixed!